src/client/get-data.js

```diff
@@ -0,0 +1,17 @@
+import { find, get } from 'lodash';
+
+export default ( clientState ) => ( endpointPath, params ) => {
+	const expandedPath = endpointPath.reduce( ( path, node ) => {
+		path.push( 'endpoints' );
+		path.push( node );
+		return path;
+	}, [] );
+	const endpoint = get( clientState, expandedPath, {} );
+
+	if ( ! params ) {
+		return endpoint.data ? endpoint.data : null;
+	}
+
+	const query = find( endpoint.queries, { params } );
+	return query && query.data ? query.data : null;
```

Comment:
This answers my prior question, so all responses from the API will be stored in `data`?

Reply:
Yes, that is correct.

Reply:
To clarify, all *valid* API responses will be stored in `data`.  If there's an error, or if it fails some kind of validation, it will still be null, or whatever the previous value was.